Formula/ignition-fuel-tools8.rb

```diff
@@ -0,0 +1,65 @@
+class IgnitionFuelTools8 < Formula
+  desc "Tools for using Fuel API to download robot models"
+  homepage "https://ignitionrobotics.org"
+  url "https://github.com/ignitionrobotics/ign-fuel-tools.git", branch: "main"
+  version "7.999.999~0~20211203"
+  license "Apache-2.0"
+
+  depends_on "cmake"
+  depends_on "ignition-cmake2"
+  depends_on "ignition-common4"
+  depends_on "ignition-msgs9"
+  depends_on "jsoncpp"
+  depends_on "libyaml"
+  depends_on "libzip"
+  depends_on macos: :high_sierra # c++17
+  depends_on "pkg-config"
+
+  def install
+    cmake_args = std_cmake_args
+    cmake_args << "-DBUILD_TESTING=Off"
+    cmake_args << "-DCMAKE_INSTALL_RPATH=#{rpath}"
+
+    mkdir "build" do
+      system "cmake", "..", *cmake_args
+      system "make", "install"
+    end
+  end
+
+  test do
+    (testpath/"test.cpp").write <<-EOS
+      #include <ignition/fuel_tools.hh>
+      int main() {
+        ignition::fuel_tools::ServerConfig srv;
+        return 0;
+      }
+    EOS
+    (testpath/"CMakeLists.txt").write <<-EOS
+      cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
+      find_package(ignition-fuel_tools7 QUIET REQUIRED)
```

Comment:
Ahh some misses here :confused: 

```suggestion
      find_package(ignition-fuel_tools8 QUIET REQUIRED)
```

Reply:
should be fixed by https://github.com/osrf/homebrew-simulation/pull/1704/commits/f31f88251caf8aeeac1c0ffbd99069a9441f3095

fix for script in https://github.com/ignition-tooling/release-tools/pull/584